openstack/placement/values.yaml

```diff
@@ -67,6 +71,32 @@ mariadb:
           enabled: true
           allTables: true
 
+pxc_db:
+  enabled: false
+  name: placement
+  alerts:
+    support_group: "compute-storage-api"
+  ccroot_user:
+    enabled: true
+  databases:
+    - placement
+  users:
+    placement:
+      name: placement
+      grants:
+        - "ALL PRIVILEGES on placement.*"
+  pxc:
+    persistence:
+      size: 10Gi
```

Comment:
Is this service-specific? How would I compute that? Any hint we can leave here for the placement chart maintainers?

Reply:
This is not service-specific, this specific value is the same as a default value in `mariadb` or new `pxc-db` chart, it's just added here explicitly to make it more transparent.